Kudu.Core/Jobs/TriggeredJobSchedule.cs

```diff
@@ -50,6 +52,8 @@ public void Reschedule(DateTime lastRun, Schedule schedule = null)
                 Logger.LogInformation("Next schedule expected in " + nextInterval);
             }
 
+            _analytics.JobEvent(TriggeredJob.Name.Fuzz(), $"Reschedule {nextInterval}", TriggeredJob.JobType, String.Empty);
```

Comment:
Is it really worth fuzzing the job name? Given that it belongs in ARM path's, it's not 'sensitive'.